vulnerable_people_form/form_pages/nhs_login_callback.py

```diff
@@ -53,4 +53,5 @@ def get_nhs_login_callback():
 
 @form.route("/start", methods=["GET"])
 def get_start():
-    return redirect("/view-or-setup")
+    session.clear()
```

Comment:
what happens when you go to the default route? As currently it goes to an error page. 

https://gds-shielded-vulnerable-people-service-staging.london.cloudapps.digital/

Reply:
It goes to a 'Page not found' page (404). We can handle this differently if required, I've not received any guidance on what the behaviour should be in this case.

Reply:
would it not go to the start of the journey ?

Reply:
I think it should probably go to the external page.

Reply:
I have spoken to Mike about this and it should:

'divert to first page of our service 'are you using this service for yourself or for someone else' page'

Reply:
Updated 🙂 Thanks!